confidant/routes/v1.py

```diff
@@ -371,6 +372,14 @@ def get_credential_list():
 @app.route('/v1/credentials/<id>', methods=['GET'])
 @authnz.require_auth
 def get_credential(id):
+    if not acl_module_check(id):
```

Comment:
I think we need to pass in more info than just the ID. I think maybe `acl_module_check('get_credential', 'view', id)`

^^ that way we can make specific checks based on function, the permission within the function, and the resource it's being called against.